pkg/controller/controller_instance_test.go

```diff
@@ -3846,7 +3848,7 @@ func TestReconcileServiceInstanceOrphanMitigation(t *testing.T) {
 				},
 			},
 			async: true,
-			finishedOrphanMitigation:     true,
+			finishedOrphanMitigation:     false,
```

Comment:
🤷‍♂️ because there was a bug in the behavior. As written in the description to PR: "The more important change is that now the OrphanMitigation condition gets reset only after we have successfully completed the orphan mitigation."